rfd/0086/compat.md

```diff
@@ -0,0 +1,7 @@
+## Backwards compatibility and support
+
+The changes in this RFD are not backwards compatible with ContainerPilot 2.x configuration. For end users using Consul as their discovery backend, each container image can be safely updated independently; there is no communication between ContainerPilot instances except through the Consul API.
```

Comment:
Probably still need some thoughts around how long we want to support ContainerPilot v2.

> We will support ContainerPilot 2.x for _N_?? (TODO)